pylint/checkers/variables.py

```diff
@@ -2666,17 +2737,19 @@ def _check_unpacking(
         values = self._nodes_to_unpack(inferred)
         if values is not None:
             if len(targets) != len(values):
-                # Check if we have starred nodes.
-                if any(isinstance(target, nodes.Starred) for target in targets):
-                    return
```

Comment:
I recognized we could move this check up higher which should help performance, and it had no negative impacts